recipe/meta.yaml

```diff
@@ -29,7 +29,7 @@ build:
   #
   # XGBoost 3 dropped CUDA 11 support.
   # xref: https://github.com/dmlc/xgboost/issues/10370
-  skip: true  # [((linux or win) and cuda_compiler_version == "None") or (python_min is undefined) or (cuda_compiler_version == "12.6") or (cuda_compiler_version == "11.8")]
+  skip: true  # [((linux or win) and cuda_compiler_version == "None") or (not python_min) or (cuda_compiler_version == "12.6") or (cuda_compiler_version == "11.8")]
```

Comment:
Just out of curiosity, what was the error that prompted this change?

Reply:
`conda-smithy` has trouble rendering the expression `python_min is undefined`. It produces error
```
INFO:conda_smithy.configure_feedstock:Applying migrations: /home/phcho/xgboost-feedstock/.ci_support/migrations/cuda128.yaml
Traceback (most recent call last):
  File "/home/phcho/miniforge3/lib/python3.12/site-packages/conda_build/metadata.py", line 392, in select_lines
    if selector_cache[selector]:
       ~~~~~~~~~~~~~~^^^^^^^^^^
KeyError: '((linux or win) and cuda_compiler_version == "None") or (python_min is undefined) or (cuda_compiler_version == "12.6") or (cuda_compiler_version == "11.8")'
```

Reply:
I applied the same fix at https://github.com/conda-forge/xgboost-feedstock/pull/238

Reply:
Interesting ok. Can see this comment: https://github.com/conda-forge/xgboost-feedstock/pull/238#discussion_r2090393946

Will see if I can figure out what is going on there

In any event this seems like a reasonable alternative approach